contrib/lib/Cargo.toml

```diff
@@ -11,14 +11,15 @@ keywords = ["rocket", "web", "framework", "contrib", "contributed"]
 license = "MIT/Apache-2.0"
 
 [features]
-default = ["json", "static_files"]
+default = ["json", "static_files", "space_helmet"]
 json = ["serde", "serde_json"]
 msgpack = ["serde", "rmp-serde"]
 tera_templates = ["tera", "templates"]
 handlebars_templates = ["handlebars", "templates"]
 static_files = []
 # Internal use only.
 templates = ["serde", "serde_json", "glob"]
+space_helmet = []
```

Comment:
I'm leaning towards calling this feature `security_headers`. I worry that `space_helmet` will make this undiscoverable.